### PR TITLE
Added Enhancement for reviews section.

### PR DESCRIPTION
**Pull request by amit-webkul**
## Issue Reference
#9972

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@amit-webkul 

Checked and found that the PR is working fine. I have checked the configurations which are added.
Checked the same in multilocale.
Fount it working, product Card, Product View Page, 

https://prnt.sc/-kFCqwLkCBtx
https://prnt.sc/HLIZjerjhkxF
https://prnt.sc/7g5HMjBbHvgC
https://prnt.sc/HLYPy9fvRB3n
https://prnt.sc/j3be9OWkMLuK
https://prnt.sc/YJa3uXT4uRGL

File: packages/Webkul/Shop/src/Resources/views/products/view/reviews.blade.php [hunk 1]
```diff
@@ -183,7 +183,12 @@ class="secondary-button items-center rounded-2xl px-8 py-2.5 max-md:w-full max-m
                                     </div>
 
                                     <p class="text-base text-zinc-500 max-sm:text-sm">
-                                        {{ $reviewHelper->getTotalRating($product) }}
+                                        {{
+                                            core()->getConfigData('catalog.products.review.summary') == 'count_of_stars' 
+                                                ? $reviewHelper->getTotalRating($product)
+                                                : $reviewHelper->getTotalReviews($product)
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view/reviews.blade.php:189]: Abstract this code in one more method and resuse everywhere.

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 2]
```diff
@@ -3560,6 +3560,9 @@
                     'review' => [
                         'allow-customer-review' => 'Allow Customer Review',
                         'allow-guest-review'    => 'Allow Guest Review',
+                        'display_review_count'  => 'Display the review count for ratings.',
+                        'display_start_count'   => 'Display the star count in ratings.',
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:3564]: Check this.

File: packages/Webkul/Shop/src/Http/Resources/ProductResource.php [hunk 3]
```diff
@@ -50,7 +50,9 @@ public function toArray($request)
             'price_html'  => $productTypeInstance->getPriceHtml(),
             'ratings'     => [
                 'average' => $this->reviewHelper->getAverageRating($this),
-                'total'   => $this->reviewHelper->getTotalRating($this),
+                'total'   => core()->getConfigData('catalog.products.review.summary') == 'count_of_stars'
+                            ? $this->reviewHelper->getTotalRating($this)
+                            : $this->reviewHelper->getTotalReviews($this),
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Resources/ProductResource.php:55]: No need of check here... simply provide one more array with total counts... i.e.

'reviews' => ['total' => ...]

File: packages/Webkul/Shop/src/Http/Resources/ProductResource.php [hunk 4]
```diff
@@ -50,7 +50,7 @@ public function toArray($request)
             'price_html'  => $productTypeInstance->getPriceHtml(),
             'ratings'     => [
                 'average' => $this->reviewHelper->getAverageRating($this),
-                'total'   => $this->reviewHelper->getTotalRating($this),
+                'total'   => $this->reviewHelper->getTotalFeedback($this),
             ],
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Resources/ProductResource.php:54]:
~~~php
...

'ratings'     => [
    'average' => $this->reviewHelper->getAverageRatings($this),
    'total'   => $this->reviewHelper->getTotalRatings($this),
],

'reviews'     => [
    'total'   => $this->reviewHelper->getTotalReviews($this),
],

...
~~~

**Review by devansh-webkul — APPROVED** [review]

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**